src/symbol/placement.js

```diff
@@ -242,7 +240,15 @@ class Placement {
             }
         }
 
-        return placementChanged;
+        // this.lastPlacementChangeTime is the time of the last commit() that
+        // resulted in a placement change -- in other words, the start time of
+        // the last symbol fade animation
+        assert(!prevPlacement || typeof prevPlacement.lastPlacementChangeTime === 'number');
```

Comment:
Why `typeof ... === 'number'` instead of just using the boolean cast of "undefined"?

Reply:
The latter would yield false for `0`... which would never happen for real `browser.now()` values, but be an annoyance in unit tests or something

Reply:
Ok fair enough. How about using `!== undefined`? I'm just looking for a form that expresses as simply as possible that this is about whether the initial value is set, not about the value possibly having multiple types.